### PR TITLE
GEODE-6633: Remove JSON pretty printing from PdxToJSON

### PR DESCRIPTION
**Pull request by jdeppe-pivotal**
Authored-by: Jens Deppe <jdeppe@pivotal.io>

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jdeppe-pivotal** [comment]
@Petahhh For your review.

**Comment by jdeppe-pivotal** [comment]
The stress tests are going to fail until GEODE-6634 is merged

File: geode-core/src/main/java/org/apache/geode/pdx/internal/json/PdxToJSON.java [hunk 1]
```diff
@@ -85,7 +84,6 @@ public String getJSON() {
   private void enableDisableJSONGeneratorFeature(JsonGenerator jg) {
     jg.enable(Feature.ESCAPE_NON_ASCII);
     jg.disable(Feature.AUTO_CLOSE_TARGET);
-    jg.setPrettyPrinter(new DefaultPrettyPrinter());
```
Comment (jinmeiliao) [inline geode-core/src/main/java/org/apache/geode/pdx/internal/json/PdxToJSON.java]: Is this the reason the json response we get for our management v2 rest call is pretty-printed?

Comment (jdeppe-pivotal) [reply geode-core/src/main/java/org/apache/geode/pdx/internal/json/PdxToJSON.java]: AFAIK this is only used for PDX queries.

**Pull request merged**